### PR TITLE
refactor: Update Service configurations for changes to common Service configuration struct

### PR DESCRIPTION
**Pull request by lenny-intel**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/master/.github/Contributing.md.

## What is the current behavior?
Using old version of ServiceInfo struct from go-mod-bootstrap

## What is the new behavior?
Using new version of ServiceInfo struct from go-mod-bootstrap

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

BREAKING CHANGE: Core/Support/SysMgmt Service configuration has changed.

## New Imports

- [ ] Yes
- [x] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

File: cmd/core-command/res/configuration.toml [hunk 1]
```diff
@@ -8,15 +8,14 @@ LogLevel = 'INFO'
...
 Host = 'localhost'
-ServerBindAddr = '' # Leave blank so default to Host value unless different value is needed.
 Port = 59882
-Protocol = 'http'
-MaxResultCount = 50000
+ServerBindAddr = '' # Leave blank so default to Host value unless different value is needed.
 StartupMsg = 'This is the Core Command Microservice'
-Timeout = 45000
+MaxResultCount = 50000
+MaxRequestSize = 0 # Not curently used.
```
Comment (jim-wang-intel) [inline cmd/core-command/res/configuration.toml:17]: what is the unit of this? KB? MB ? it should be helpful if put the unit on the comment if not on the key itself.

File: internal/support/scheduler/schedule.go [hunk 2]
```diff
@@ -506,8 +506,12 @@ func execute(
 			lc.Error("create new request occurs error : " + err.Error())
 		}
 
+		// Note this is V1 code that will be removed soon.
+		// Change is so code will compile, but not worried about error checking.
+		timeout, _ := time.ParseDuration(configuration.Service.RequestTimeout)
+
```
Comment (jim-wang-intel) [inline internal/support/scheduler/schedule.go:512]: nice comment so we know why error is ignored!

File: cmd/core-command/res/configuration.toml (hunk 1 above)
Comment (lenny-intel) [reply cmd/core-command/res/configuration.toml]: Added comment to each file.

**Review by jim-wang-intel — APPROVED** [review]

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**